policies/templates/gcp_storage_location_v1.yaml

```diff
@@ -53,7 +53,7 @@ spec:
     validation.gcp.forsetisecurity.org:
       rego: | #INLINE("validator/storage_location.rego")
           #
-          # Copyright 2018 Google LLC
+          # Copyright 2019 Google LLC
```

Comment:
is this change meant to be part of this PR?

Reply:
It's a side effect of make format / make build